docs(edge-services): typo and minor fixes MTA-7306 (#6683)
scaleway/docs-content: pages/edge-services/reference-content/cname-record.mdx

|----------------------------|-------------------------------|
| `videos.example.com`       | `otherdomain.com`             |

> In this case, when a DNS server sees this record for `videos.example.com` it will know not to direct traffic to `videos.example.com`'s own IP address, but to that of `otherdomain.com`. It will find `otherdomain.com`'s IP address via its [A record](/domains-and-dns/reference-content/understanding-dns-records/#a-record).

Reviewer: ```suggestion
In this case, when a DNS server sees this record for `videos.example.com`, it does not to direct traffic to `videos.example.com`'s own IP address, but to that of `otherdomain.com`. It finds `otherdomain.com`'s IP address via its [A record](/domains-and-dns/reference-content/understanding-dns-records/#a-record).
```

When the client actually connects to `otherdomain.com`'s IP address, the web server can see that the requested URL was `videos.example.com`, and deliver the relevant content.

## When and why do I need to create a CNAME record for Edge Services?

<Message type="note">
    You only need CNAME records if you want to customize the standard Edge Services endpoint.

> </Message>

Author: ```suggestion
    You only need CNAME records if you want to customize the standard Edge Services endpoint.
```

>     If you do not want to customize the standard Edge Services endpoint, you do not need CNAME records.

Reviewer 2: - J'ai un doute sur le fait que le callout soit le format approprié quand on répond à une question qui se trouve en heading, du texte simple pourrait suffire non ? Est-ce qu'on répond à du feedback utilisateur / des soucis côté support avec ce changement ? @firdevs-a ton avis m'intéresse également :) 
- Dans la mesure du possible, comme dans l'UX Writing, je préfère éviter les tournures négatives, surtout répétées. Penses-tu qu'on puisse appliquer ici ? (e.g., "You only need CNAME records if you want to customize the standard Edge Services endpoint." / "CNAME records are required to customize the standard Edge Services endpoint ; you do not need them otherwise")

Reviewer: En effet, ici la note est inutile et une tournure positive serait plus appréciable. La phrase que tu proposes va très bien: You only need CNAME records if you want to customize the standard Edge Services endpoint.

Reviewer 2: @Loic-kd il ne reste plus qu'à faire sauter les balises de la note pour obtenir mon approval 😄

>     If you do not want to customize the standard Edge Services endpoint, you do not need CNAME records.

Author: ```suggestion
    You only need CNAME records if you want to customize the standard Edge Services endpoint.
```

> When you create an Edge Services pipeline to a [backend](/edge-services/concepts/#backend), the backend content is served through the standard Edge Services endpoint, e.g. `pipeline-id.svc.edge.scw.cloud`. If you want to [customize your Edge Services endpoint with your own subdomain](/edge-services/how-to/configure-custom-domain/), you must create a CNAME record to point your subdomain to the Edge Services endpoint. 

Reviewer: ```suggestion
When you create an Edge Services pipeline to a [backend](/edge-services/concepts/#backend), the backend content is served through the standard Edge Services endpoint, e.g., `pipeline-id.svc.edge.scw.cloud`. If you want to [customize your Edge Services endpoint with your own subdomain](/edge-services/how-to/configure-custom-domain/), you must create a CNAME record to point your subdomain to the Edge Services endpoint. 
```

>   - If your domain is managed with [Scaleway Domains and DNS](/domains-and-dns/quickstart/), Scaleway will generate the appropriate CNAME record. It will be visible among your Domains and DNS records in the console. Do not modify or delete the CNAME record. If you deactivate Edge Services, Scaleway will automatically delete the CNAME record. 

Reviewer: ```suggestion
  - If your domain is managed with [Scaleway Domains and DNS](/domains-and-dns/quickstart/), Scaleway generates the appropriate CNAME record. It is visible among your Domains and DNS records in the console. Do not modify or delete the CNAME record. If you deactivate Edge Services, Scaleway automatically deletes the CNAME record. 
```

>   - If your domain is managed with [Scaleway Domains and DNS](/domains-and-dns/quickstart/), Scaleway will generate the appropriate CNAME record. It will be visible among your Domains and DNS records in the console. Do not modify or delete the CNAME record. If you deactivate Edge Services, Scaleway will automatically delete the CNAME record. 

Author: ```suggestion
  - If your domain is managed with [Scaleway Domains and DNS](/domains-and-dns/quickstart/), Scaleway generates the appropriate CNAME record during the Edge Services pipeline creation. It is visible among your Domains and DNS records in the console. Do not modify or delete the CNAME record. If you deactivate Edge Services, Scaleway automatically deletes the CNAME record.
```

>   - If your domain is managed by an external provider, Scaleway cannot create the appropriate CNAME record. You will be prompted, as part of the process for customizing your Edge Services domain, to create this record yourself with your domain provider.

Reviewer: ```suggestion
  - If your domain is managed by an external provider, Scaleway cannot create the appropriate CNAME record. You are prompted, as part of the process for customizing your Edge Services domain, to create this record yourself with your domain provider.
```
